packages/web-app-files/src/components/AppBar/SharesNavigation.vue

```diff
@@ -56,17 +61,21 @@ export default {
       return routes
     }, {})
     const navItems = computed(() => [
+      // computed aka not reactive?
```

Comment:
dev leftover? computed -> immutable (at least not via setting a value)

Reply:
Meh, that was my reminder to investigate what later "magically" worked 🤷🏽 removing it